unit_tests/src/test_zone_report.cpp

```diff
@@ -60,6 +59,6 @@ TEST_CASE("json zone report")
                 id_str = resource["id"].get<std::string>();
             }
         }
-        REQUIRE(info->id() == id_str);
+        REQUIRE(info.id() == id_str);
```

Comment:
I am confused by this change since all the other `adm::client::resource_info`s are treated as `std::optional`, but this one is not. Can you explain?

Reply:
I don't see that in the final result.

I'm guessing you're looking at each commit individually?

Reply:
Indeed